modules/ROOT/pages/provisioning-vmware.adoc

```diff
@@ -155,3 +156,31 @@ VM_NAME='fcos-node01'
 govc device.serial.add -vm "${VM_NAME}"
 govc device.serial.connect -vm "${VM_NAME}" "[datastore] ${VM_NAME}/console.log"
 ----
+
+== Modifying OVA metadata
+
+NOTE: While we provide these instructions for modifying the OVA, we cannot
+guarantee that any modifications to the OVA metadata will result in a usable
+guest VM.
+
+Fedora CoreOS ships the VMware OVA with certain metadata defaults in the OVA file.
+The hardware version and default firmware are of particular interest, as they
+can affect how the guest VM operates in the hypervisor. These defaults may
+prevent the use of FCOS as a guest VM in older versions of VMware products and
+it may be necessary to modify the metadata in order to use FCOS as a guest VM.
+
+The VMware OVA is simply a tarball that contains the files `disk.vmdk` and
+`coreos.ovf`. In order to edit the metadata used by FCOS as a guest VM, you
+should untar the OVA artifact, edit the OVF file, then create a new OVA file.
```

Comment:
In principle the OVA should also include a `.mf` file, which is a list of SHA digests for the other files.  We're not doing that because when I tried, I got mysterious hash mismatches I couldn't track down.  I don't think it's especially important, but wanted to note that these instructions assume the user doesn't need to update or remove an `.mf` file after changing the `.ovf`.